pkglistgen/tool.py

```diff
@@ -785,7 +895,23 @@ def update_and_solve_target(
                 for line in sorted(output):
                     f.write(line + '\n')
 
-        self.commit_package(product_dir)
+        if git_url:
+            if product_dir and os.path.isdir(product_dir):
+                logging.debug(subprocess.check_output(
+                    ['git', 'add', product_dir],
+                    cwd=cache_dir, encoding='utf-8'))
+            if os.path.isdir(self.productcompose_dir):
+                logging.debug(subprocess.check_output(
+                    ['git', 'add', self.productcompose_dir],
+                    cwd=cache_dir, encoding='utf-8'))
+
+            logging.debug(subprocess.check_output(
+                'git commit -m "Update by pkglistgen of openSUSE-release-tool"', cwd=cache_dir, shell=True, encoding='utf-8'))
```

Comment:
why does that use "shell=True" ? simply split it into an iterable (like tuple, or list like above)?

Reply:
changed that in additional commit